subsys/profiler/profiler_nordic.c

```diff
@@ -149,6 +149,9 @@ int profiler_init(void)
 		k_sched_unlock();
 		return 0;
 	}
+	for (size_t i = 0; i < CONFIG_PROFILER_MAX_NUMBER_OF_EVENTS; i++) {
```

Comment:
This changes previous default behavior (previously if shell was enabled, profiling of events was disabled by default). Please update documentation if needed, add information to changelog.